ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -716,6 +716,12 @@ const NetworksForm = ({
     !rpcUrl ||
     !chainId ||
     !ticker;
+  let displayRpcUrl = rpcUrl?.includes(`/v3/${infuraProjectId}`)
+    ? rpcUrl.replace(`/v3/${infuraProjectId}`, '')
+    : rpcUrl;
+  if (viewOnly) {
+    displayRpcUrl = displayRpcUrl.toLowerCase();
```

Comment:
The logic above suggests the `rpcUrl` can be missing hence the `?` but this logic assumes it always calls `toLowerCase`.

Can we remove the optional chaining or do we need to add it here also?

Reply:
To be on safe side, let me add it here also.